cdisc_rules_engine/sql_operations/sql_base_operation.py

```diff
@@ -62,12 +71,17 @@ def execute(self) -> SqlOperationResult:
             logger.debug(f"error in operation {self.__class__.__name__}: {str(e)}")
             raise
         except Exception as e:
-            # Log unexpected errors
+            # Log unexpected errors and wrap with operation context
             logger.error(
                 f"error in operation {self.__class__.__name__}: {str(e)}",
                 exc_info=True,
             )
-            raise
+            # Convert class name to operation name (e.g., NumericOperation -> numeric)
+            operation_name = self.__class__.__name__.lower()
+            if operation_name.endswith("operation"):
```

Comment:
Don't bother stripping this stuff, doing things like this always makes everything a little more fragile, we can just ignore it when reading